doc/user/upgrade/upgrade_guide.md

```diff
@@ -22,6 +22,30 @@ $ kubectl edit deployment/etcd-operator
 ### Incompatible upgrade
 In the case of an incompatible upgrade, the process requires restoring a new cluster from backup. See the [incompatible upgrade guide](incompatible_upgrade.md) for more information.
 
+## v0.6.0 -> v0.6.1
+
+**Note:** If your cluster does not have the following backup policy then you can simply upgrade the operator to the `v0.6.1` image.
+
+Backup policy:
+- A PV storage type: `backup.StorageType=PersistentVolume`
+- No storage class specified: `backup.pv.StorageClass` is unset
+
+In releases prior to `v0.6.1` the etcd-operator would by default create a new storage class for the above backup policy. The `Provisioner` field for that storage class would be set to the vaule of the `--pv-provisioner` flag(`kubernetes.io/gce-pd` or `kubernetes.io/aws-ebs`).
+
+In the `v0.6.1` release the etcd-operator will no longer create a storage class by default. This behavior is set by the new flag `--create-storage-class` whose default value is `false`.
+
+So if your cluster has the above backup policy then do the following steps before upgrading the operator image to `v0.6.1`.
```

Comment:
Put this right after mentioning the backup policy before mentioning the flag.
I feel mentioning the flag should be better put in CHANGELOG?

Reply:
The flags and their values are already explained in the CHANGELOG. Over here I've just mentioned the `create-storage-class` flag again to give some context for the change that is needed.